# HG changeset patch
# User Rik <rik@octave.org>
# Date 1522429386 25200
#      Fri Mar 30 10:03:06 2018 -0700
# Branch stable
# Node ID 6c1611e6830f78960f377434d24f4039c3a8d744
# Parent  6652d3823428e479ace872dcc4f8863dbbbf97e5
settings-dialog.ui: Correct typo "curent" -> "current" (bug #53526).

* settings-dialog.ui: Correct typo "curent" -> "current".

diff --git a/libgui/src/settings-dialog.ui b/libgui/src/settings-dialog.ui
--- a/libgui/src/settings-dialog.ui
+++ b/libgui/src/settings-dialog.ui
@@ -1163,17 +1163,17 @@
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="toolTip">
                    <string>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</string>
                   </property>
                   <property name="text">
-                   <string>Wrap long lines at curent window border</string>
+                   <string>Wrap long lines at current window border</string>
                   </property>
                   <property name="checked">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
